docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -77,7 +77,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
     ![Run and Debug](media/with-visual-studio-code/rundebug.png)
```

Comment:
given how the blue arrow and red square are now highlighted in this image, it'd probably be better to move this down to step 7 then.

Reply:
I have moved it to step 7.
However, there's also a highlight around the debug-sidebar to show locals and watches etc. Is it OK to keep the images as-is?

Reply:
Yes, let's keep it like that and if we get any complaints, we can fix it.